cmd/frcli/node_audit.go

```diff
@@ -13,10 +13,40 @@ import (
 	"github.com/lightninglabs/faraday/frdrpc"
 )
 
+var splitter = "/regex/"
+
 var onChainReportCommand = cli.Command{
 	Name:     "audit",
 	Category: "reporting",
 	Usage:    "Get a report of node activity.",
+	Description: fmt.Sprintf(`
+	Create a report containing all of your node's activity over 
+	the period specified. Transactions are sorted into a set of 
+	lightning-specific entry types. Fiat values can optionally be 
+	included using the --enable_fiat flag. To write the report 
+	directly to a csv, set a target directory using the --csv_path 
+	flag.
+
+	These reports can optionally be created with custom categories. 
+	This requires providing a name for the category, and a set of 
+	regular expressions which identify the labels of transactions 
+	belonging in the category. To directly string match, just provide
+	the string itself. These regular expressions are matched against 
+	the labels for on chain transactions and the invoices on memos 
+	(at present we cannot match forwarding events and payments).
+
+	Each category is specified as follows, using /regex/ to delimit
+	sections (because regular expressions themselves contain most
+	regular delimiters):
+	name:%vregularexpression%vregularexpression
+
+	To specify a single category provide the following: 
+	--categories=name:%vregularexpression}%vregularexpression
+
+	Multiple categories need to be wrapped in braces and comma 
+	separated, but may *not* contain any spaces
+	--categories={categorystring,categorystring}
+`, splitter, splitter, splitter, splitter),
```

Comment:
nit: can use `%[1]v` to always use the the same argument (the number in the bracket is the one-based index).

Reply:
TIL!